paimon-common/src/main/java/org/apache/paimon/CoreOptions.java

```diff
@@ -1053,6 +1053,22 @@ public class CoreOptions implements Serializable {
                                                             + STATS_MODE_SUFFIX))
                                     .build());
 
+    public static final ConfigOption<Boolean> METADATA_STATS_DENSE_STORE =
+            key("metadata.stats-dense-store")
+                    .booleanType()
+                    .defaultValue(false)
```

Comment:
Why the default value is not true?
You are worry about that many users are using old versions of Paimon sdk in their reading engine?

Reply:
Yes, it is true